tests/e2e/utils/prometheus/IAMPolicyUtils.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+import json
+from e2e.utils.utils import get_aws_account_id
+
+IAM_CLIENT = boto3.client(service_name='iam')
+
+# Will create a permission policy if there is no existing policy with the given policy name.    
+def create_policy_if_not_exist(policy_name, permission_policy_file_name):
+    policy_arn = f'arn:aws:iam::{get_aws_account_id()}:policy/{policy_name}'
+    try:
+        IAM_CLIENT.get_policy(PolicyArn=policy_arn).get('Policy').get('Arn')
+    except:
+        print("Creating a new policy named", policy_name, "since there was no existing policy by this name.")
+        
+        with open(f'{permission_policy_file_name}.json') as permission_policy_file:
+            permission_policy = json.dumps(json.load(permission_policy_file))
+        policy_arn = IAM_CLIENT.create_policy(
+            PolicyName=policy_name,
+            PolicyDocument=permission_policy
+        ).get('Policy').get('Arn')
+    return policy_arn
+
+# Will try to delete the given IAM policy.
+def delete_IAM_policy(policy_name):
+    policy_arn = f'arn:aws:iam::{get_aws_account_id()}:policy/{policy_name}'
+    print("About to delete policy with arn {policy_arn}")
```

Comment:
nit: maybe instead of About to use Attempting to since it may fail here as the rest of the code demonstrates.